scalafmt-core/shared/src/main/scala/org/scalafmt/util/TreeOps.scala

```diff
@@ -703,7 +704,7 @@ object TreeOps {
     }
 
   // Redundant {} block around case statements
-  def isCaseBodyABlock(ft: FormatToken, caseStat: Tree): Boolean = {
+  def isCaseBodyABlock(ft: FormatToken, caseStat: CaseTree): Boolean = {
```

Comment:
should we have added the body def to CaseTree, using the common type, since they are different? would have simplified this code, too.

Reply:
Ach, yeah I can fix that.

Reply:
https://github.com/scalameta/scalameta/pull/2212

Reply:
4.4.6 ready to go?

Reply:
Yep! Took me a while today to get it working for 2.12.13, but it's finally up.